src/oidcc_token.erl

```diff
@@ -1010,32 +1009,41 @@ validate_jwt(Jwt, ClientContext, Opts) when is_map(Opts) ->
 
     case {SigningAlgs, EncryptionAlgs} of
         {[], []} ->
-            error(badarg, [Jwt, ClientContext, Opts], []);
+            error(badarg, [Token, ClientContext, Opts], []);
         _ ->
             ok
     end,
 
-    ExpClaims = [{<<"iss">>, Issuer}],
     Jwks1 =
         case ClientJwks of
             none -> Jwks0;
             #jose_jwk{} -> oidcc_jwt_util:merge_jwks(Jwks0, ClientJwks)
         end,
     Jwks2 = oidcc_jwt_util:merge_client_secret_oct_keys(Jwks1, SigningAlgs, ClientSecret),
     Jwks = oidcc_jwt_util:merge_client_secret_oct_keys(Jwks2, EncryptionAlgs, ClientSecret),
-    TrustedAudience = maps:get(trusted_audience, Opts, any),
+    TrustedAudience = maps:get(trusted_audiences, Opts, any),
```

Comment:
```suggestion
    TrustedAudiences = maps:get(trusted_audiences, Opts, any),
```

And can we make the same change in `verify_aud_claims/3`? Just to clarify that it is a list and not a single binary. 🙏

Reply:
Updated.